spyderlib/qt/__init__.py

```diff
@@ -9,7 +9,7 @@
 
 import os
 
-os.environ.setdefault('QT_API', 'pyqt')
+os.environ.setdefault('QT_API', 'pyqt5')
```

Comment:
I don't know what to think about this one. Our support for PyQt5 is still experimental, and there aren't packages on Windows for Anaconda and WinPython.